### PR TITLE
Add PostgreSQL JDBC driver

### PR DESCRIPTION
**Pull request by mwperina**
Adds PostgreSQL JDBC driver due to recent regression by PostgreSQL JDBC
drivers > 42.2.10

Bug-Url: https://bugzilla.redhat.com/2077794
Signed-off-by: Martin Perina <mperina@redhat.com>

### CODE REVIEW

File: ovirt-dependencies.spec.in [hunk 1]
```diff
@@ -22,6 +22,8 @@ BuildRequires:  tar
 
 Provides:   bundled(gwt-servlet) = @GWT_VERSION@
 
+Provides:   bundled(posttgresql-jdbc) = @POSTGRESQL_JDBC_VERSION@
```
Comment (dupondje) [inline ovirt-dependencies.spec.in]: Typo? posttgresql

**Review by sandrobonazzola — CHANGES_REQUESTED** [review]
See @dupondje comment

File: ovirt-dependencies.spec.in (hunk 1 above)
Comment (arso) [reply ovirt-dependencies.spec.in]: +1

**Review by arso — CHANGES_REQUESTED** [review]
Looks good except a typo in spec 'posttgresql' . 
I was only a bit surprised that xmvn did not complain about missing parent pom ... perhaps it's already shipped with different package?

EDIT: actually it did complain.  
Unable to resolve parent POM org.postgresql:pgjdbc-core-parent:pom:1.1

These 2 parent poms need to be added under %install and above postgresqljdbc:
%mvn_artifact dependencies/org/postgresql/pgjdbc-versions/1.1.6/pgjdbc-versions-1.1.6.pom
%mvn_artifact dependencies/org/postgresql/pgjdbc-core-parent/1.1.6/pgjdbc-core-parent-1.1.6.pom

File: ovirt-dependencies.spec.in [hunk 2]
```diff
@@ -54,6 +56,8 @@ export JAVA_HOME="/usr/lib/jvm/java-11-openjdk"
 %mvn_artifact dependencies/com/google/gwt/gwt/@GWT_VERSION@/gwt-@GWT_VERSION@.pom
 %mvn_artifact dependencies/com/google/gwt/gwt-servlet/@GWT_VERSION@/gwt-servlet-@GWT_VERSION@.pom dependencies/com/google/gwt/gwt-servlet/@GWT_VERSION@/gwt-servlet-@GWT_VERSION@.jar
 
+%mvn_artifact dependencies/org/postgresql/postgresql/@POSTGRESQL_JDBC_VERSION@/postgresql-@POSTGRESQL_JDBC_VERSION@.pom dependencies/org/postgresql/postgresql/@PPOSTGRESQL_JDBC_VERSION@/postgresql-@POSTGRESQL_JDBC_VERSION@.jar
```
Comment (arso) [inline ovirt-dependencies.spec.in]: typo: @PPOSTGRESQL_JDBC_VERSION@   -> @POSTGRESQL_JDBC_VERSION@

**Comment by arso** [comment]
I have pushed updates (I've made them for the review anyway)

File: ovirt-dependencies.spec.in (hunk 2 above)
Comment (arso) [reply ovirt-dependencies.spec.in]: Fixed

File: ovirt-dependencies.spec.in (hunk 1 above)
Comment (arso) [reply ovirt-dependencies.spec.in]: fixed

**Review by arso — APPROVED** [review]

**Review by sandrobonazzola — APPROVED** [review]
LGTM

**Comment by mwperina** [comment]
I'd wait with merging till Monday, we should know if this workaround is needed or we will have better solution

**Comment by mwperina** [comment]
Closing in favor of https://github.com/oVirt/ovirt-engine/pull/322 and https://github.com/oVirt/ovirt-dependencies/pull/10

**Pull request closed without merging**